src/dhalsim.js

```diff
@@ -1,16 +1,61 @@
 const defaultDhalsimOptions = {
   breakingPointValue: '',
+  tablet: '768px',
+  desktop: '960px',
+  wide: '1200px',
 };
 
 export default class Dhalsim {
   constructor(options) {
     const defaultOptions = Object.assign({}, defaultDhalsimOptions, options);
     this.breakingPointValue = defaultOptions.breakingPointValue;
 
+    this.tablet = defaultOptions.tablet;
+    this.desktop = defaultOptions.desktop;
+    this.wide = defaultOptions.wide;
```

Comment:
This assignments are not necessary, because `Object.assign` already assign this properties, with `defaultDhalsimOptions`.

Reply:
Sorry, `Object.assign`, line 10, are not assigning `this`. We could make some thing directly like this: 

```JavaScript
Object.assign(this, defaultDhalsimOptions, options);
```

And delete const `defaultOptions` =D